src/mmw/js/src/modeling/models.js

```diff
@@ -53,48 +63,71 @@ var ProjectModel = Backbone.Model.extend({
         this.set('name', newName);
     },
 
-    saveAll: function() {
-        var self = this;
-
-        // If we haven't saved the project before,
-        // hold on to the scenarios so that we can
-        // save each one once we have an ID for the project
-        if (this.isNew) {
-            var tempScenarios = this.get('scenarios');
+    saveProjectAndScenarios: function() {
+        if (App.user.get('uid') !== this.get('uid')) {
+            return;
         }
```

Comment:
Yeah but it creates a lot of unnecessary noise on the server and in the logs. Why let requests we know will fail go through.